jekyll/_cci2/faq.md

```diff
@@ -290,11 +299,15 @@ An `active user` is any user who triggers the use of compute resources on non-OS
 
 **Note:** If your project is [open-source]({{ site.baseurl }}/2.0/oss) you will **not** be considered an active user.
 
+To find a list of your Active User's, Log in to the CircleCI web app > click `Settings` > click `Plan Usage` > click on the `Users` tab.
+
 #### What happens when I run out of credits?
 {:.no_toc}
 
 On the Performance plan, when you reach 10% of your remaining credits, you will
-be refilled 25% of your credits. For example, If your monthly package size is 25,000 credits, you will automatically be refilled 6,250 credits when you reach 2,500 remaining credits.
+be refilled 25% of your credit subscription (at 25% the cost of your plan). For example, If your monthly package size is 25,000 credits, you will automatically be refilled 6,250 credits when you reach 2,500 remaining credits.
+
+If you notice that your account is receiving repeated refills, you can review your credit usage by navigating to the settings tab of your account and clicking on “Plan Usage."
```

Comment:
We should add something about how to "optimize" your plan.

Reply:
So, this would be a way to avoid the repeated refills with a scenario

Reply:
can you also add the price?

"refilled 6,250 credits (at $.0006 each)"

Reply:
Fixed!